packages/core/protocols/src/proto/dxos/config.proto

```diff
@@ -174,123 +145,111 @@ message Runtime {
   // KUBE
   //
   message Kube {
-    message Autoupdate {
-      bool enabled = 1;
-      /// Seconds
-      int32 interval = 2;
-    }
+    // TODO: "bin" / "kernel" / "core" / "runtime" / "system"?
+    message System {
+      message Autoupdate {
+        bool enabled = 1;
+        /// Seconds
+        int32 interval = 2;
+      }
 
-    message Https {
-      bool enabled = 1;
-      string port = 2;
-      string email = 3;
-      string certfile = 4;
-      string keyfile = 5;
-      string dnsprovider = 6; // TODO(burdon): dns_provider.
-    }
+      Autoupdate autoupdate = 1;
 
-    message P2P {
-      // TODO(egorgripasov): private_key
-      string privatekey = 1;
-      string port = 2;
-      repeated string bootstrap = 3;
+      /// Environment variables.
+      repeated string env = 100;
     }
 
-    message Monitoring {
-      bool enabled = 1;
-      string endpoint = 2;
-      /// Seconds
-      int32 interval = 3;
-    }
+    message Instrumentation {
+      /// Monitoring OTEL configuration.
+      message Monitoring {
+        bool enabled = 1;
+        string endpoint = 2;
+        /// Seconds
+        int32 interval = 3;
+      }
+  
+      /// Sentry tracing configuration.
+      message Trace {
+        bool disabled = 1;
+      }
+  
+      /// Segment telemetry configuration.
+      message Telemetry {
+        bool disabled = 1;
+      }
+  
+      message Gravity {
+        bool disabled = 1;
+        string name = 2;
+        optional string log_dir = 3;
+      }
 
-    message Trace {
-      bool disabled = 1;
+      Monitoring monitoring = 1;
+      Trace trace = 2;
+      Telemetry telemetry = 3;
+      Gravity gravity = 4;
     }
 
-    message Telemetry {
-      bool disabled = 1;
-    }
+    message Server {
+      message Https {
+        bool enabled = 1;
+        int32 port = 2;
+        string email = 3;
+        string cert_file = 4;
+        string key_file = 5;
+        string dns_provider = 6;
+      }
 
-    message Gravity {
-      bool disabled = 1;
-      string name = 2;
-      optional string log_dir = 3;
-    }
+      string host = 1;
+      int32 port = 2;
+
+      /// Hostname to use in KUBE configuration response.
+      string host_in_config = 3;
+
+      /// Alises for the hostname (e.g. dev.dxos.org vs dev.kube.dxos.org)
+      repeated string host_alias = 4;
 
-    message AppTunneling {
-      bool enabled = 1;
-      string command_template = 2;
-      string output_re = 3;
-      string port = 4;
+      Https https = 5;
     }
 
-    message Bots {
-      bool enabled = 1;
-      /// Refresh interval - Seconds
-      int32 interval = 2;
+    message Services {
+      message Signal {
+        string private_key = 1;
```

Comment:
this is a configuration piece of a signal bootstrap node;
we have to use a persistent key, otherwise each our bootstrap node will change it's address after each restart